src/tests.rs

```diff
@@ -749,10 +786,14 @@ async fn exports_imported_earlier_default() {
     ],
   )
   .await;
-  let entries = DocParser::new(&graph, false, analyzer.as_capturing_parser())
-    .unwrap()
-    .parse_with_reexports(&specifier)
-    .unwrap();
+  let entries = DocParser::new(
+    &graph,
+    analyzer.as_capturing_parser(),
+    DocParserOptions::default(),
+  )
+  .unwrap()
+  .parse_with_reexports(&specifier)
+  .unwrap();
```

Comment:
Something for a follow up - add a helper that will set it all up and hide `unwrap`s

Reply:
Yeah, I think a lot of these tests could be moved to the specs folder